src/error.rs

```diff
@@ -139,6 +143,13 @@ pub enum ErrorKind {
     /// please report it to the warpgrapher team.
     FieldMissingResolverError(String, String),
 
+    /// Returned by the `From` implementation for a `String` object.
+    /// Conversion from `String`.
+    GenericError(String),
```

Comment:
This will be a rough experience for developers using the framework.  I would argue against the existence of a GenericError.  Errors are never generic. There's always something specific that triggers them.

Reply:
This has been removed